Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -78,15 +114,21 @@
     'vtPositives': 10,
     'vtPositiveUrlsForIP': 30
 }
-# The dictionary below does not represent DBot Scores correctly, and should not be used
-dbotscores = {
-    'Critical': 4,
-    'High': 3,
-    'Medium': 2,
-    'Low': 1,
-    'Unknown': 0,
-    'Informational': 0.5
-}
+
+
+class IndicatorType:
+    """
+    Enum: contains all the indicator types, used DBotScore.Type)
+    """
+    IP = 'ip'
```

Comment:
suggests merging my branch here https://github.com/demisto/content/pull/5368
Already added more indicators type (that needed for TIP integrations).
You should preserve the name of this class as its singular.